.github/workflows/website-integrity.yml

```
@@ -0,0 +1,46 @@
+name: Website Integrity
+on:
+  pull_request:
+    paths:
+      - 'pages/**'
+    branches:
+      - main
+jobs:
+  website-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Fetch
+        run: git fetch origin main
+
+      - name: Setup env
+        uses: the-guild-org/shared-config/setup@main
+        with:
+          nodeVersion: 18
+          packageManager: yarn
+
+      - name: Build Site
+        run: yarn build
+
+      - name: Compare
+        run: git diff origin/${{ github.base_ref }}.. -- route-lockfile.txt
+
+      - name: Diff to file
+        if: always()
+        id: diff_result
+        run: |
+          OUTPUT=$(git diff origin/${{ github.base_ref }}.. -- route-lockfile.txt)
+          OUTPUT="${OUTPUT//'%'/'%25'}"
+          OUTPUT="${OUTPUT//$'\n'/'%0A'}"
+          OUTPUT="${OUTPUT//$'\r'/'%0D'}"
+          echo "::set-output name=result::$OUTPUT"
```

```suggestion
          echo "result<<EOF" >> $GITHUB_OUTPUT
          echo "$(git diff origin/${{ github.base_ref }}.. -- route-lockfile.txt)" >> $GITHUB_OUTPUT
          echo EOF >> $GITHUB_OUTPUT
```

set-output is deprecated